lib/arboreal/instance_methods.rb

```diff
@@ -69,6 +69,7 @@ def ancestor_conditions
     end
 
     def descendant_conditions
+      return ["1 = 0"] if new_record?
```

Comment:
Agreed - I realized we hadn't added a spec last night as I was thinking through this. Thanks for the heads up on the other areas to check!